app/analytics.go

```diff
@@ -53,7 +53,9 @@ func (a *App) GetAnalytics(name string, teamId string) (model.AnalyticsRows, *mo
 		if teamId == "" {
 			userInactiveChan = a.Srv.Store.User().AnalyticsGetInactiveUsersCount()
 		} else {
-			userChan = a.Srv.Store.User().AnalyticsUniqueUserCount(teamId)
+			userChan = a.Srv.Store.User().Count(model.UserCountOptions{
```

Comment:
back in the day @lieut-data pushed me towards using "functional options" for UplaodFile. I did not become a convert, but I do wonder if we should have a consistent practice.

Reply:
Heh, I can't quite remember what my thoughts were on this at the time. @levb, can you remind me what you think might be a more consistent practice here?

Reply:
k, so first of all, I am not proposing any changes here. I think it's both intention-revealing, and typographically good. 

My personal preference for optional parameters is to have fewer functions, and a simple struct of options, with meaningful defaults. 

I only asked because I would like us to (eventually!) work out a "MM golang best practices" doc, for things like this.

Reply:
100% agree that we should formalize our pattern here, and to lean towards fewer functions and better defaults. Worth a developer's meeting discussion?